admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -396,6 +396,13 @@ Nested Groups:
   By default a group will only contain users, so enabling this option isn't necessary.
   However, if groups are contained inside groups, and this option is not enabled, any groups contained within other groups will be ignored and not returned in search results.
 
+  In Active Directory you have to enter this specific filter in order for the ownCloud LDAP app to be able to find users in nested groups.
+  
+  Example:
+
+  * ``memberof:1.2.840.113556.1.4.1941:=CN=<Group_Name>,dc=example,dc=com``
```

Comment:
It would be good to explain what these elements in the example mean so the admin can derive from that the parameters for his environment

Reply:
I lack knowledge to tell you what those numbers mean. As far as I know it's just code that the AD interprets as member of nested group.